lighthouse-core/fraggle-rock/config/default-config.js

```diff
@@ -11,12 +11,14 @@ const {deepClone} = require('../../config/config-helpers.js');
 /** @type {LH.Config.AuditJson[]} */
 const frAudits = [
   'byte-efficiency/uses-responsive-images-snapshot',
+  'metrics/interaction-to-next-paint',
 ];
 
 /** @type {Record<string, LH.Config.AuditRefJson[]>} */
 const frCategoryAuditRefExtensions = {
   'performance': [
     {id: 'uses-responsive-images-snapshot', weight: 0},
+    {id: 'interaction-to-next-paint', weight: 30, group: 'metrics', acronym: 'INP'},
```

Comment:
this is the same weight as TBT so it's 50/50, but maybe we don't care because timespan reports don't show the weighted score?

Reply:
If this is going to be a diagnostic in navigation mode, it might be helpful to make the score 0, but someone reading the config wouldn't know that it's still important for timespans.

Reply:
> If this is going to be a diagnostic in navigation mode, it might be helpful to make the score 0, but someone reading the config wouldn't know that it's still important for timespans.

yeah, makes sense, done. The scoring is confusing at the JSON level in timespans, but scoring is confusing in timespans, so that's nothing new :)